i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/messages.mdx

```diff
@@ -330,3 +330,56 @@ Wenn Sie Famedly auf Ihrem Desktop oder mobil verwenden, können Sie Nachrichten
 | -   Aufzählungsliste  | Fügen Sie ein Sternchen und ein Leerzeichen vor Ihrem Text ein.                     |
 
 * * *
+
+## Zu Notizen hinzufügen
+
+Sie können Nachrichten und Medien aus anderen Unterhaltungen speichern, indem Sie sie zu <b>Notizen</b> hinzufügen.
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ol>
+    <li>Klicken Sie auf die Nachricht, die Sie speichern möchten.</li>
+    <li>Wählen Sie nun <b>Zu Notizen hinzufügen</b> aus der Liste aus.</li>
+    <li>Die Nachricht wird in einem privaten Chat namens <b>Notizen</b> gespeichert.</li>
+    <li>Sie können auf die Schaltfläche <b>Zu Notizen gehen</b> klicken, die am unteren Rand des Fensters erscheint (und nach einigen Sekunden verschwindet), um nachzusehen.</li>
+  </ol>
+    </TabItem>
+    <TabItem value="mobile" label="Mobile">
+  <ol>
+    <li>Tippen Sie auf die Nachricht, die Sie speichern möchten.</li>
+    <li>Wählen Sie nun <b>Zu Notizen hinzufügen</b> aus der Liste aus.</li>
+    <li>Die Nachricht wird in einem privaten Chat namens <b>Notizen</b> gespeichert.</li>
+    <li>Sie können auf die Schaltfläche <b>Zu Notizen gehen</b> klicken, die am unteren Rand des Bildschirms erscheint (und nach einigen Sekunden verschwindet), um nachzusehen.</li>
+  </ol>
+  </TabItem>
+</Tabs>
+
+:::info
+
+Der **Notizen**-Chat wird erstellt und wird erst sichtbar, nachdem Sie Ihre erste Nachricht mit der Aktion **Zu Notizen hinzufügen** gespeichert haben.
+
+:::
+
+## Sich selbst Nachrichten schicken
+
+Neben dem Speichern von Nachrichten oder Medien aus anderen Gruppen- oder Privatchats können Sie diesen Chat verwenden, um Nachrichten und Medien zu senden. Aber nur Sie haben Zugriff darauf.
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ol>
+    <li>Stellen Sie sicher, dass Sie im Abschnitt "Chats" sind.</li>
+    <li>Klicken Sie auf den oben angepinnten <b>Notizen</b>-Chat.</li>
```

Comment:
For the DAU or people that don't read info boxes, please add:
`<li>Falls Sie den <b>Notizen</b>-Chat nicht in Ihrer Chat-Liste finden können, fügen Sie bitte wie oben beschrieben eine Nachricht zu den Notizen hinzu, um den Chat zu erstellen.</li>`